packages/lib/modules/swap/SwapProvider.tsx

```diff
@@ -199,6 +205,7 @@ export function _useSwap({ poolActionableTokens, pool, pathParams }: SwapProvide
       tokenOut: swapState.tokenOut.address,
       swapType: swapState.swapType,
       swapAmount: getSwapAmount(),
+      poolIds: getPoolIds(),
```

Comment:
Please add a comment to make the filter usage explicit. For example: 
"We only use this field to filter by the specific pool swap path when swapping from a concrete pool"